debezium-core/src/main/java/io/debezium/connector/common/BaseSourceTask.java

```diff
@@ -213,6 +223,10 @@ private void updateLastOffset(Map<String, ?> partition, Map<String, ?> lastOffse
      */
     protected abstract List<SourceRecord> doPoll() throws InterruptedException;
 
+    public boolean shouldRestartOnRetriableException() {
```

Comment:
Would it be possible to move the method to `ErrorHandler`? It would probably belong logically there.